### PR TITLE
Info for SMP/e PTFs included in the installation guide

### LINKED ISSUE #1193: v1.11.LTS does not describe PTFs
## Description

The user guide for v1.11.LTS on zowe.org still says
```
Since this is the first release of the SMP/E package, no PTFs have been created.
```

## Pages to Update

User guide PDF and zowe.org web page

## Expected behavior

Doc should list PTFs by release and describe how to apply them using SMP/E

## Additional context

The zowe.org downloads page does not explicitly say that users who do not already have the FMID must download and install it before installing the PTF - although any sysprog will know that.  
So a new user will have to click two separate buttons.

It might also be useful to add a brief note that any Zowe PTF rolls up all the previous ones, so users should install the latest one and older ones are not needed.

### PR DESCRIPTION
**Pull request by Joe-Winchester**
#### Is there a related issue for this PR? 
fixes #1193

### CODE REVIEW

File: docs/user-guide/install-zowe-smpe.md [hunk 1]
```diff
@@ -124,7 +124,14 @@ All issues of previous releases of Zowe that were resolved before August 2019 ha
 
 ### Service level information
 
-Since this is the first release of the SMP/E package, no PTFs have been created.
+The Zowe SMP/E package is a distribution of Zowe version 1.9.0 with an FMID of AZWE001.
+
+Subsequent releases of the Zowe z/OS components are delivered as rollup PTFs and delivered on [zowe.org](https://zowe.org#download).  Because of the file size of the PTF  it is packaged as two co-requisites PTFs which are made available in a single .zip download.
```
Comment (John-A-Davies) [inline docs/user-guide/install-zowe-smpe.md:129]: Should say `co-requisite` i.e. singular.

File: docs/user-guide/install-zowe-smpe.md [hunk 2]
```diff
@@ -124,7 +124,14 @@ All issues of previous releases of Zowe that were resolved before August 2019 ha
...
 ### Service level information
 
-Since this is the first release of the SMP/E package, no PTFs have been created.
+The Zowe SMP/E package is a distribution of Zowe version 1.9.0 with an FMID of AZWE001.
+
+Subsequent releases of the Zowe z/OS components are delivered as rollup PTFs and delivered on [zowe.org](https://zowe.org#download).  Because of the file size of the PTF  it is packaged as two co-requisites PTFs which are made available in a single .zip download.  
+
+Zowe release | PTF 1 | PTF 2 
+---|---|---
+[1.10](https://d1xozlojgf8voe.cloudfront.net/legal.html?type=smpe&version=1.10.0) | UO01938 | UO01940
```
Comment (John-A-Davies) [inline docs/user-guide/install-zowe-smpe.md:133]: 1st PTF is 1939, not 1938.

File: docs/user-guide/install-zowe-smpe.md (hunk 1 above)
Comment (nannanli) [reply docs/user-guide/install-zowe-smpe.md]: Fixed this while editing the description.

File: docs/user-guide/install-zowe-smpe.md (hunk 2 above)
Comment (nannanli) [reply docs/user-guide/install-zowe-smpe.md]: Fixed this while editing the description.

**Pull request merged**